docling/backend/msword_backend.py

```diff
@@ -230,6 +235,7 @@ def _walk_linear(
                                     parent=self.parents[level - 1],
                                     name="shape-text",
                                 )
+                                added_elements.append(shape_group.get_ref())
                                 doc.add_text(
                                     label=DocItemLabel.PARAGRAPH,
```

Comment:
Can we use `TEXT` instead of `PARAGHRAPH`. I would like to deprecate `PARAGHRAPH`.